core/controllers/base_test.py

```diff
@@ -1105,11 +973,8 @@ def test_error_response_for_get_request_of_type_json_has_json_format(self):
         fake_urls.append(main.get_redirect_route(r'/fake', self.FakeHandler))
         fake_urls.append(main.URLS[-1])
         with self.swap(main, 'URLS', fake_urls):
-            transaction_services = models.Registry.import_transaction_services()
-            app = transaction_services.toplevel_wrapper(  # pylint: disable=invalid-name
```

Comment:
This is not required for Python 3 apps?

Reply:
No, it is not, it is replaced by creating a separate context for each request (https://github.com/oppia/oppia/pull/13395/files/586319bbbea63066eeec8fd3bd3afb95cb2863aa#diff-b10564ab7d2c520cdd0243874879fb0a782862c3c902ab535faabe57d5a505e1R924)